Code-the-Dream-School/bb-practicum-team-3-front
# 404 page

## Description


Add 404 page.  

Also in this pull request I also have a custom mui theme in which boxSizing is border-box and the body margin is 0. Body doesn't seem to be applied.

## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


![image](https://user-images.githubusercontent.com/74852114/233909965-a002a08a-4394-4676-aa64-765de1c50682.png)


### After



## Testing Steps / QA Criteria
Navigate to a path in which we don't have routes for. Should take you to the 404 page. Click the "go back" button to navigate to the last page/url path you were on
